xbmc/input/actions/ActionIDs.h

```diff
@@ -381,18 +382,23 @@ constexpr const int ACTION_TOGGLE_DIGITAL_ANALOG = 202; //!< switch digital <->
 constexpr const int ACTION_RELOAD_KEYMAPS = 203; //!< reloads CButtonTranslator's keymaps
 constexpr const int ACTION_GUIPROFILE_BEGIN = 204; //!< start the GUIControlProfiler running
 
-constexpr const int ACTION_TELETEXT_RED = 215; //!< Teletext Color button <b>Red</b> to control TopText
-constexpr const int ACTION_TELETEXT_GREEN = 216; //!< Teletext Color button <b>Green</b> to control TopText
-constexpr const int ACTION_TELETEXT_YELLOW = 217; //!< Teletext Color button <b>Yellow</b> to control TopText
-constexpr const int ACTION_TELETEXT_BLUE = 218; //!< Teletext Color button <b>Blue</b> to control TopText
+constexpr const int ACTION_TELETEXT_RED =
+    215; //!< Teletext Color button <b>Red</b> to control TopText
+constexpr const int ACTION_TELETEXT_GREEN =
+    216; //!< Teletext Color button <b>Green</b> to control TopText
+constexpr const int ACTION_TELETEXT_YELLOW =
+    217; //!< Teletext Color button <b>Yellow</b> to control TopText
+constexpr const int ACTION_TELETEXT_BLUE =
+    218; //!< Teletext Color button <b>Blue</b> to control TopText
```

Comment:
Personally im not a fan of the changes in ActionIDs.h
I think it makes things less readable, i think we would be better off doing clang format off/on for this block.

But whatever people want is fine, just thought id voice my opinion.

Reply:
Easy solution, we comment like a normal person would:

```c++
//!< Teletext Color button <b>Red</b> to control TopText
constexpr const int ACTION_TELETEXT_RED = 215;

//!< Teletext Color button <b>Green</b> to control TopText
constexpr const int ACTION_TELETEXT_GREEN = 216;

//!< Teletext Color button <b>Yellow</b> to control TopText
constexpr const int ACTION_TELETEXT_YELLOW = 217;

//!< Teletext Color button <b>Blue</b> to control TopText
constexpr const int ACTION_TELETEXT_BLUE = 218;
```

Thanks for taking a look. I didn't put as much work into this PR as the mongo savestate manager one. I'll update the code.

Reply:
also very acceptable, just looked like more work, haha

Reply:
clang made pretty bad work out of the pvr inclues too. I went through the entire PR by hand, should be good now.

Reply:
We change 180 lines now down from 460